### ISSUE TITLE
Access token should not be saved in UserDefaults

### ISSUE THREAD

**Issue by jasonnam** [muzix/nimblesurveys#7]
Access token should be saved in Keychain as it is a credential.

**Comment by muzix** [comment]
Yes, agree. I used UserDefault here for fast prototype. I think I can do better here by implementing the adapter interface for saving key value data, then make the UserDefault comform to it by using Extension. Then I can write another implementation comform to that interface which use Keychain store under the hood.
